src/caret_analyze/plot/timeseries/response_time_timeseries.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021 Research Institute of Systems Planning, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from collections.abc import Sequence
+
+import pandas as pd
+
+from ..metrics_base import MetricsBase
+from ...record import RecordsInterface, ResponseTime
+from ...runtime import Path
+
+
+class ResponsetimeTimeSeries(MetricsBase):
+    """Class that provides latency timeseries data."""
+
+    def __init__(
+        self,
+        target_objects: Sequence[Path],
+        case: str
+    ) -> None:
+        super().__init__(target_objects)
+        self._case = case
+
+    def to_dataframe(self, xaxis_type: str = 'system_time') -> pd.DataFrame:
+        timeseries_records_list = self.to_timeseries_records_list(xaxis_type)
+        all_df = pd.DataFrame()
+        for to, response_records in zip(self._target_objects, timeseries_records_list):
+            response_df = response_records.to_dataframe()
+            response_df[response_df.columns[-1]] *= 10**(-6)
+            response_df.rename(
+                columns={
+                    response_df.columns[0]: 'path_start_timestamp [ns]',
+                    response_df.columns[1]: 'response_time [ms]',
+                },
+                inplace=True
+            )
+            # TODO: Multi-column DataFrame are difficult for users to handle,
+            #       so it should be a single-column DataFrame.
+            response_df = self._add_top_level_column(response_df, to)
+            all_df = pd.concat([all_df, response_df], axis=1)
+
+        return all_df.sort_index(level=0, axis=1, sort_remaining=False)
+
+    def to_timeseries_records_list(
```

Comment:
Please add CARET_doc.

Ref: https://github.com/tier4/CARET_analyze/blob/a9a0c424d27b1dee2323584ecb46fbfa132547c0/src/caret_analyze/plot/timeseries/frequency_timeseries.py#L80-L95

Reply:
Fixed in https://github.com/tier4/CARET_analyze/commit/f4cd2af6e1f243a8c68186f3341424cb97eae919